browser_windows.go

```diff
@@ -1,16 +1,13 @@
+//go:generate mkwinsyscall -output zbrowser_windows.go browser_windows.go
+//sys ShellExecute(hwnd int, verb string, file string, args string, cwd string, showCmd int) (err error) = shell32.ShellExecuteW
 package browser
 
-import (
-	"os/exec"
-	"strings"
-	"syscall"
-)
+import "os/exec"
+const SW_SHOWNORMAL = 1
 
 func openBrowser(url string) error {
-	r := strings.NewReplacer("&", "^&")
-	return runCmd("cmd", "/c", "start", r.Replace(url))
+   return ShellExecute(0, "", url, "", "", SW_SHOWNORMAL)
```

Comment:
Is this. available in the x/sys package? If not, would they accept this function so we don't have to own it.

Reply:
It is available:

https://pkg.go.dev/golang.org/x/sys/windows#ShellExecute

but personally I dont like the signature. They went with `*uint16`, when `string` is the closer match, and with the `mkwinsyscall` tool, really no reason to have to put the conversion on the end user.

Reply:
I would prefer to use the version x/sys and to work with the maintainers of that package to adjust the signature if they agree.

Reply:
Understand. I did a PR on `x/sys` for a similar issue https://github.com/golang/sys/pull/96 I think they are on code freeze until February though. I was going to wait to see how that panned out before taking further action. Regarding this repo, as a stopgap I can just switch it to the package version if thats preferable.

Reply:
Thanks for looking at that. I’ll merge this as is and let’s come back when 1.16 is out and see how much can be up streamed into x/sys